spec/automated_review/id_columns_are_bigint_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'set'
+describe "*_id columns" do
+  EXCEPTIONS_LIST = Set.new(
+    [
+      "authentications.ldap_id",
+      "authentications.rhsm_pool_id",
+      "cloud_networks.provider_segmentation_id",
+      "container_volumes.common_volume_id",
+      "miq_queue.task_id",
+      "network_ports.binding_host_id",
+      "scan_histories.task_id",
+      "sessions.session_id"
+    ]
+  ).freeze
+
+  it "should be type bigint" do
+    ActiveRecord::Base.connection.tables.each do |table|
+      next if ManageIQ::Schema::SYSTEM_TABLES.include?(table)
+      DatabaseHelper.columns_for_table(table).each do |column|
```

Comment:
Moving forward, perhaps this should move into a DatabaseHelper method...

```ruby
def DatabaseHelper.each_table
  ActiveRecord::Base.connection.tables.each do |table|
    next if ManageIQ::Schema::SYSTEM_TABLES.include?(table)
    yield table
  end
end

def DatabaseHelper.each_column
  each_table do |table|
    columns_for_table(table).each do |column|
      yield table, column
    end
  end
end
```

Then caller just does

```ruby
DatabaseHelper.each_column do |table, column|
   ...
end
```